### DISCUSSION TITLE
Remove unused references

### DISCUSSION THREAD

**Discussion by regesguitar** [microsoft/react-native-windows#11205] [General]
How can I remove unused references in my UWP RNW (React Native for Windows) project? I tried using the "Remove unused references" option in Visual Studio, but it's not available. I also installed the Visual Studio Code Cleaner plugin, but it's not working. Can someone help me find a solution to this problem?

Current result screenshot: 

![image](https://user-images.githubusercontent.com/50219913/218105318-dbad7748-ee60-4796-a9e9-ba8795b5ec3c.png)

Expected result screenshot:
![image](https://user-images.githubusercontent.com/50219913/218106616-030a9322-c86d-46b0-ba15-a3f332ed8afb.png)

**Comment by caiovictor** [comment]
A while ago I also needed this feature and I couldn't use it, unfortunately I had to do it manually. 
But it's a good question why this doesn't work with RNW projects. :bulb:

**Comment by regesguitar** [comment]
Can anybody help me ? @alloy @kant @Thristhart
